spack/packages/dla-future/package.py

```diff
@@ -46,7 +46,7 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
     conflicts("umpire@6:")
 
     depends_on("pika +mpi")
-    depends_on("pika@0.8:")
+    depends_on("pika@0.9:")
```

Comment:
I'm slightly losing track of what versions are required for what. Is this bump necessary or just nice to have here? To be clear, I don't mind the bump here even it's not strictly necessary, just trying to keep track of the why.

Reply:
I think if there's no need to restrict the version unless really required.

Reply:
> I mean in any case we have a fixed version for pika in the CMakeLists.txt, so I figured since there is no release for DLAF, it should be exactly 0.9 but I don't mind the column

I read this too quickly earlier. We _don't_ have a fixed version in `CMakeLists.txt` (there's no `EXACT`). However, see my other comment on https://github.com/eth-cscs/DLA-Future/pull/657.

Reply:
This comment: https://github.com/eth-cscs/DLA-Future/pull/657#discussion_r999371105.

Reply:
TL;DR: pika 0.9.0 _is_ required for these changes (for HIP, but we can require it for everything).